src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/ScreenRecorder.cs

```diff
@@ -366,7 +445,7 @@ private void ProcessFrames()
                     ImageConversion.EncodeNativeArrayToJPG(frameData, graphicsFormat, (uint)width, (uint)height);
 
                 // write out the image to file
-                var path = $"{directoryPath}/{frameNumber}".PadLeft(9, '0') + ".jpg";
+                var path = $"{directoryPath}_screenshots/{frameNumber}".PadLeft(9, '0') + ".jpg";
```

Comment:
I would make these subfolders , so that we don't have 2x the director listings in the top level folder.. 1 directory listing per run, inside.. screenshots and data

Reply:
Fixed!